webapp/crashstats/settings/base.py

```diff
@@ -433,9 +464,9 @@ def filter(self, record):
     }
 }
 
-TIME_ZONE = config("TIME_ZONE", "UTC")
+TIME_ZONE = config("TIME_ZONE", default="UTC")
```

Comment:
This is weird. We never set the `TIME_ZONE` so I have no idea why this was configurable.

Reply:
I would imagine this ensures that we use UTC in dev configurations, which is good for consistency

Reply:
We don't set this in server environments, so it should always be `UTC`. It's curious.